docs/design/uplink-scopes.md

```diff
@@ -52,7 +52,7 @@ Note that the encrypted path does not need to match the unencrypted path, and th
 
 ## Rationale
 
-This proposal is not a radical departure of the current method, just an extension to allowing a collection of paths to compose a bucket at any unencrypted spot. Thus, it shoudn't require any changes on the server side, reducing risk and allowing for earlier shipping. For example, any current configuration
+This proposal is not a radical departure of the current method, just an extension to allowing a collection of paths to compose a bucket at any unencrypted spot. Thus, it shoudn't require any changes on the server side, reducing risk and allowing for earlier shipping. For example, any current configuration can be represented as a scope with a default project key and no bucket shares.
```

Comment:
I would interpret "default project key" as using the `nil` value for `ProjectKey`, but that's probably not what you mean. What does default mean here?

Reply:
"can be represented as a scope with a `ProjectEncryptionKey` specified and no bucket shares."?

Reply:
Oh I get it, it was treating `ProjectKey` as a default in concept (because it gets used when no matching BucketShare is available). Ok. Yeah your suggested wording would be easier for me to understand.